packages/core/src/accordion/accordion.stories.ts

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import '@clr/core/accordion/register.js';
+import { getElementStorybookArgTypes, spreadProps, getElementStorybookArgs } from '@clr/core/internal';
+import { action } from '@storybook/addon-actions';
+import { html } from 'lit-html';
+import customElements from '../../dist/core/custom-elements.json';
+
+export default {
+  title: 'Components/Accordion/Stories',
+  component: 'cds-accordion',
+  argTypes: getElementStorybookArgTypes('cds-accordion-panel', customElements),
+  parameters: {
+    options: { showPanel: true },
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/file/v2mkhzKQdhECXOx8BElgdA/Clarity-UI-Library---light-2.2.0?node-id=1007%3A0',
+    },
+  },
+};
+
+export const API = (args: any) => {
+  return html`
+    <cds-accordion>
+      <cds-accordion-panel
+        ...="${spreadProps(getElementStorybookArgs(args))}"
+        @expandedChange=${action('expandedChange')}
+      >
+        <cds-accordion-header>${args['cds-accordion-header']}</cds-accordion-header>
+        <cds-accordion-content>${args['cds-accordion-content']}</cds-accordion-content>
+      </cds-accordion-panel>
+    </cds-accordion>
+  `;
+};
+
+export const basicAccordion = () => {
+  return html`
+    <cds-accordion>
+      <cds-accordion-panel expanded>
+        <cds-accordion-header>Item 1</cds-accordion-header>
+        <cds-accordion-content>Content 2</cds-accordion-content>
+      </cds-accordion-panel>
+      <cds-accordion-panel disabled>
+        <cds-accordion-header>Item 2</cds-accordion-header>
+        <cds-accordion-content>Content 2</cds-accordion-content>
+      </cds-accordion-panel>
+      <cds-accordion-panel>
+        <cds-accordion-header>Item 3</cds-accordion-header>
+        <cds-accordion-content>Content 3</cds-accordion-content>
+      </cds-accordion-panel>
+    </cds-accordion>
+  `;
+};
+
+export const customStyles = () => {
+  return html`
+    <style>
+      cds-accordion-panel.app-custom {
+        --border-color: midnightblue;
+        --header-color: aliceblue;
```

Comment:
Could we include changing the color of the caret/angle icon here? I think it would be helpful for people to see/know they can do that.